src/ui-config/marketsConfig.tsx

```diff
@@ -44,16 +45,15 @@ export type MarketDataType = {
 };
 
 export enum CustomMarket {
-  // v3 test networks
+  // v3 test networks, all v3.0.1 with permissioned faucet
   proto_arbitrum_goerli_v3 = 'proto_arbitrum_goerli_v3',
   proto_mumbai_v3 = 'proto_mumbai_v3',
   proto_fantom_testnet_v3 = 'proto_fantom_testnet_v3',
   proto_fuji_v3 = 'proto_fuji_v3',
   proto_goerli_v3 = 'proto_goerli_v3',
   proto_optimism_goerli_v3 = 'proto_optimism_goerli_v3',
-  // v3.0.1 mainnets
-  proto_mainnet_v3_1 = 'proto_mainnet_v3_1',
   // v3 mainnets
+  proto_mainnet_v3 = 'proto_mainnet_v3',
```

Comment:
@defispartan do you think we should name `proto_mainnet_v3_1` so we know the difference between v3 & v3.0.1

Reply:
I kept it as v3 because once other v3 markets are updated, the titles won't need to be manually updated, which may also break other things if the market title is checked directly in other places

Reply:
ok